### PR TITLE
Add housing.com

### PR DESCRIPTION
**Pull request by stevedomin**
https://twitter.com/HousingEngg/status/715502677623353344

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ A curated list of companies using Elixir in production, organized by industry.
 #### Real Estate
 
 * [Opendoor](https://www.opendoor.com) - Opendoor buys your home from you, so you can skip months of selling and start your next chapter.
+* [Housing](https://housing.com/) - Housing.com is an Indian online real estate listing portal that simplifies finding, renting, and selling homes in India.
```
Comment (doomspork) [inline README.md]: Could you alphabetize this please?

**Comment by doomspork** [comment]
Thank you @stevedomin!  Would you mind updating the change to be in :abc: order?

File: README.md (hunk 1 above)
Comment (stevedomin) [inline README.md]: Done!

**Comment by stevedomin** [comment]
Sorry, didn't notice it was ordered

**Comment by doomspork** [comment]
No problem, thank you @stevedomin :+1:

**Pull request merged**